### PR TITLE
Auth plugin experiment

### PR DESCRIPTION
**Pull request by DianeHu**
* Auth working, on succeed proceeds to next page
* Saving authtoken, additional functionality not implemented rn (fetching access/refresh token)
* TODO: on failure should pop up modal with explanation of error?

### CODE REVIEW

File: aux_ui/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLIENT_ID=316d39c75e9c4e13a93c937f6f6b7477
```
Comment (j3remych3n) [inline aux_ui/.env:1]: spoop

File: aux_ui/lib/screens/link_spotify.dart [hunk 2]
```diff
@@ -4,16 +4,73 @@ import 'package:aux_ui/theme/aux_theme.dart';
...
 import 'package:aux_ui/widgets/buttons/icon_bar_button.dart';
+import 'package:spotify_sdk/spotify_sdk.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+//import 'package:spotify_sdk/models/crossfade_state.dart';
+//import 'package:spotify_sdk/spotify_sdk.dart';
+//import 'package:spotify_sdk/models/player_state.dart';
+//import 'package:spotify_sdk/models/player_context.dart';
+//import 'package:spotify_sdk/models/connection_status.dart';
+//import 'package:spotify_sdk/models/image_uri.dart';
```
Comment (j3remych3n) [inline aux_ui/lib/screens/link_spotify.dart:15]: delet pls

File: aux_ui/.env (hunk 1 above)
Comment (DianeHu) [reply aux_ui/.env]: Removed cached env, shouldn't be tracked anymore!

File: aux_ui/lib/screens/link_spotify.dart (hunk 2 above)
Comment (DianeHu) [reply aux_ui/lib/screens/link_spotify.dart]: Is delete

**Pull request merged**